README.md

```diff
@@ -36,12 +41,13 @@ Options:
 ## Example
 
 ``` shell
-npm install -g puppeteer-cli
+npm install -g puppeteer-v-cli
 echo "<h1>Hello world!</h1>" > mypage.html
-puppeteer print mypage.html myprint.pdf # local file
-puppeteer print https://github.com/JarvusInnovations/puppeteer-cli puppeteer-cli.pdf # url
-puppeteer screenshot mypage.html myscreenshot.png # local file
-puppeteer screenshot https://jarv.us myscreenshot.png # url
+puppeteer-v print mypage.html myprint.pdf # local file
+puppeteer-v print https://github.com/JarvusInnovations/puppeteer-cli puppeteer-cli.pdf # url
+puppeteer-v screenshot mypage.html myscreenshot.png # local file
+puppeteer-v screenshot https://jarv.us myscreenshot.png # url
+puppeteer-v screenshot https://jarv.us myscreenshot.png --viewport-width 100 --viewport-height 100 
```

Comment:
drop the `-v` everywhere, and consider my suggested syntax which for a square like this would be `--viewport 100`